build.xml

```diff
@@ -10,12 +10,14 @@
     <target name="compile">
         <mkdir dir="build/classes"/>
         <javac srcdir="jnius/src" destdir="build/classes"
+               encoding="utf-8"
                includeantruntime='false' />
     </target>
 
     <target name="test-compile">
         <mkdir dir="build/test-classes"/>
         <javac srcdir="tests/java-src" destdir="build/test-classes"
+               encoding="utf-8"
```

Comment:
Out of the context of this PR.
This should be addressed via https://github.com/kivy/pyjnius/pull/525 so I may simply rebase once it gets merged

Reply:
Now rebased this change should no longer appear :)

Reply:
fix merged in master :)